src/routes_admin.py

```diff
@@ -0,0 +1,237 @@
+from flask import render_template, abort, request, jsonify
+import json
+
+from src import db, util, auth, bw_api
+from src.common import verify_staff, verify_admin, verify_student
+
+
+
+class AdminRoutes:
+    def __init__(self, blueprint):
+        def none_modified(result):
```

Comment:
Add some comments to this, so it is clear what inputs it is taking, and explaining why we need this.

Reply:
A comment on the classes (StaffRoutes, StudentRoutes, AdminRoutes), this is not the best way to use blueprints. We should create a blueprint in each of those files (e.g. AdminRoutes = BluePrint('admin_routes', __name__, template_folder='templates/admin')), then import these files in the __init__.py, and register each of those blue print. Reference  https://flask.palletsprojects.com/en/1.1.x/blueprints/

I think we should refactor all these files this way.

Reply:
Sounds like a good idea, but let's postpone the major refactor to a separate PR.